frontend/src/components/Footer/footer.module.scss

```diff
@@ -151,6 +152,9 @@
   color: #c2d6d7;
 }
 
+.nav-ul li {
+  text-align: center;
+  
```

Comment:
closing paranthesis (`}`) is missing here

Reply:
yes this is the reason of build faliure.